WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeWebViewFragment.java

```diff
@@ -118,6 +121,17 @@ public void onResume() {
         setNavigationIcon(R.drawable.ic_close_white_24dp);
     }
 
+    private void clearCookies() {
```

Comment:
Am I correct in thinking `removeAllCookies()` will affect every WebView in the app? If so, this *might* be a problem since it would force the user to re-login to any authenticated page they visited in the app.

Also, keep in mind that clearing cookies is asynchronous, so it's possible that the cookies won't be cleared before the page is browsed. If this is a concern, you may want to [flush](http://developer.android.com/intl/es/reference/android/webkit/CookieManager.html#flush()) the cookie manager before browsing (keeping in mind that's a blocking call).

One final nit: this seems like it belongs in a utility class since it's not tied to the publicize feature.